src/components/Nav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import logo from './Assets/weather-app.png';
+import continents from './Assets/continents.png';
+
+const Nav = () => (
+  <>
+    <nav className="navbar">
+      <img className="logo" src={logo} alt="logo" />
+      <h1>AirVibe</h1>
+    </nav>
+    <header className="map-header">
+      <div className="map">
+        <img src={continents} alt="Asia Map" />
+      </div>
+      <h1>Air Quality in Global Cities</h1>
+    </header>
+  </>
```

Comment:
- [ ] I would suggest a better markup for this heading component:

Since everything represents the header of your app, I suggest something like this:

```js
<header>
  <nav>
    <img className="logo" src={logo} alt="logo" />
    <h1>AirVibe</h1>
  </nav>
  <div clasName="map-header">
    <div className="map">
      <img src={continents} alt="Asia Map" />
    </div>
    <h2>Air Quality in Global Cities</h2> // Notice this change from h1 to h2
  </div>
</header>
```

Having a semantic markup matters for better SEO, accessibility, and cleaner HTML. Remember that is a good practice to only have one `<h1>` tag per page. Please consider applying this change!

Reply:
This should be optional